lib/tasks/make_region_plants.rake

```diff
@@ -65,12 +65,12 @@ namespace :make_region_plants do
             Plant.where("scientific_name like ?", "%#{array[3]}%").each do |f|
```

Comment:
Indentations in this file seem to be off, seen from here: https://github.com/cchchoi1986/hikehk/blob/regional/lib/tasks/make_region_plants.rake

Make sure that the tab size = 2